python/py-grpcio/Portfile

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-grpcio
+version             1.10.0
+platforms           darwin
+license             Apache-2
+maintainers         {@emcrisostomo gmail.com:enrico.m.crisostomo} openmaintainer
+
+description         HTTP/2-based RPC framework
+long_description    ${description}
+
+homepage            https://grpc.io/
+master_sites        pypi:g/grpcio
+distname            ${python.rootname}-${version}
+
+checksums           rmd160 ecbc517bdf994012605b9d698ef45fd94d68aab8 \
+                    sha256 03265472d39bf26f124c3ef68446f7873c8260893e6ae65b323a5b51ed52e580 \
+                    size   13987692
+
+python.versions     27 35 36
+
+if {${name} ne ${subport}} {
+    depends_build-append    \
+        port:py${python.version}-cython \
+        port:py${python.version}-setuptools
+
+    depends_lib-append      \
+        port:openssl \
```

Comment:
Yes, I will

Reply:
Is that actually a standard at this point?

Reply:
All ports do it. Does that constitute a standard?

Reply:
All ports do not, in fact, do it.

Reply:
And your constant insistence that OpenSSL is not safe and badgering people into supporting LibreSSL for builds even if it isn't tested is starting to get irritating. It's one thing to say "I think this would be better", but you're pretending there's been a policy decision and that there's something wrong with the library that most upstreams specify and use, and that's disingenuous.

Reply:
> All ports do not, in fact, do it.

I stand corrected: there is nine ports that don't.